src/matlab.ts

```diff
@@ -67,7 +67,22 @@ export async function runCommand(hs: HelperScript, platform: string, fn: ExecFn)
  * @param platform Operating system of the runner (e.g., "win32" or "linux")
  */
 export function getRunMATLABCommandScriptPath(platform: string): string {
-    const ext = platform === "win32" ? "bat" : "sh";
-    const rmcPath = path.join(__dirname, "bin", `run_matlab_command.${ext}`);
+    let ext;
+    let platform_dir;
+    switch (platform) {
+        case "win32":
+            ext = ".exe";
+            platform_dir = "win64";
+            break;
+        case "darwin":
+            ext = "";
+            platform_dir = "maci64";
+            break;
+        default:
+            ext = "";
+            platform_dir = "glnxa64";
```

Comment:
There's a bit of a disconnect here between platform and architecture. Maybe this function should take in the architecture as well so we can be sure we're picking the right binary or erroring if we don't have a binary for that combination.

Reply:
Hmmm yeah that's a good point. Is there any architectures beyond x64 that we know to be in use currently? Could this technically be a "breaking change" for some self-hosted runners?

Reply:
It looks like GitHub only [supports x64 runners (and ARM32/64 on Linux)](https://docs.github.com/en/actions/hosting-your-own-runners/about-self-hosted-runners#supported-architectures-and-operating-systems-for-self-hosted-runners), so we should be safe providing just x64 binaries. I think just from a code clarity and correctness standpoint, we should still take the arch into consideration here and just error if it doesn't match our expectations.